package.json

```diff
@@ -44,17 +44,17 @@
     "@types/node": "^15.0.2",
     "@types/semver": "^7.3.5",
     "@typescript-eslint/eslint-plugin": "^4.26.0",
-    "@typescript-eslint/parser": "^4.22.1",
-    "@vercel/ncc": "^0.27.0",
+    "@typescript-eslint/parser": "^4.26.0",
```

Comment:
Okay then, I will revert the bumps. I started with the bump because one dependency `@typescript-eslint/parser` was not in sync with the other package `@typescript-eslint/eslint-plugin`.